### ISSUE TITLE
Helping localizations

### ISSUE THREAD

**Issue by bellingeri** [angryip/ipscan#239]
Hi.

Trying to contribute to the Italian translation, I always have doubts about the presence of new strings to translate.
The web tools provided by GitHub, in my opinion, are not made for this purpose.

To simplify the work of translators it would be useful to use tools such as transifex.com, localise.biz, crowdin.com, poeditor.com or something similar.

@angryziber: what do you think?

Thanks in advance.

**Comment by angryziber** [comment]
@bellingeri good idea, does any of them support Java ResourceBundles or property files?

**Comment by bellingeri** [comment]
@angryziber: I have always used these tools as a translator, but each one has help pages on the .properties files:
https://docs.transifex.com/formats/java-properties
https://localise.biz/help/formats/importing/java
https://support.crowdin.com/supported-formats/
https://poeditor.com/help/how_to_translate_java_properties

transifex.com has a specific plan for open source projects: https://docs.transifex.com/projects/open-source-project

crowdin.com also: https://crowdin.com/page/open-source-project-setup-request

poeditor.com also: https://poeditor.com/kb/open-source-localization

localise.biz is free up to 2000 translations: https://localise.biz/plans
